lib/simple_segment/operations/operation.rb

```diff
@@ -36,7 +36,8 @@ def base_payload
           context: context,
           integrations: options[:integrations],
           timestamp: timestamp(options.fetch(:timestamp, current_time)),
-          sentAt: current_time.iso8601
+          sentAt: current_time.iso8601,
+          messageId: options[:message_id]
```

Comment:
The [Analytics for Ruby](https://github.com/segmentio/analytics-ruby/blob/169e99b22264e3bd48e5ad54582cfc29276cbf38/lib/segment/analytics/field_parser.rb#L138) library does a `to_s` explicit coercion here. But I noticed we aren't doing that coercion for any other id fields, so I didn't do that. Let me know if you think of a reason I should.

Reply:
Let's leave it as is (without `to_s`) for now. The [docs](https://segment.com/docs/connections/sources/catalog/libraries/server/ruby/) currently show it as Time, which is likely a typo 😆 

Do you know if its ok to send a `nil` message_id in the payload? Looks like the python client [generates a random uuid](https://github.com/segmentio/analytics-python/blob/cfa025eeebfae611cf37045628e7ec0df59c3a8f/analytics/client.py#L253-L254), but the ruby client seems to be sending nils 🤔

Reply:
Did a quick test and current behavior works as expected, we can adopt the python client behavior in the future if needed (PRs are welcome!).